network/router_test.go

```diff
@@ -204,8 +205,10 @@ func testRouterAutoConnection(t *testing.T, fac routerFactory) {
 		t.Fatal("Should be able to stop h2")
 	}
 	err = h1.Send(h2.ServerIdentity, &SimpleMessage{12})
+	if err == nil {
+		t.Error("h1 let us send still")
+	}
 	require.NotNil(t, err)
```

Comment:
Why not
```
require.NotNil(t, err, "h1 let us send still")
```
?

Reply:
It was left over from debugging. I do not like how require prints its errors, because Emacs cannot jump to the line. Can we just agree that sometimes calling t.Error(f) is ok?

Reply:
OK for me.